package.json

```diff
@@ -202,16 +204,16 @@
   "size-limit": [
     {
       "path": "commonjs/index.js",
-      "limit": "212 KB"
+      "limit": "350 KB"
```

Comment:
I'm wondering if we can omit the original package from the deps after generating icons? 

The main question here is whether we can actually get tree-shaking if we still are importing the blueprint icons in `Icon`.